src/main/resources/application.properties

```diff
@@ -23,4 +23,8 @@ scheduler.cdc.queue.ttlSeconds=\${SCHEDULER_CDC_QUEUE_TTL_SECONDS}
 scheduler.cdc.queue.visibilityTimeoutWalletCdc=\${SCHEDULER_CDC_QUEUE_VISIBILITY_TIMEOUT_SECONDS}
 # Retry policy for sending message
 scheduler.cdc.retry-send.maxAttempts=\${SCHEDULER_CDC_SEND_RETRY_MAX_ATTEMPTS}
-scheduler.cdc.retry-send.intervalInMs=\${SCHEDULER_CDC_SEND_RETRY_INTERVAL_IN_MS}
\ No newline at end of file
+scheduler.cdc.retry-send.intervalInMs=\${SCHEDULER_CDC_SEND_RETRY_INTERVAL_IN_MS}
+# CDC resume policy
+cdc.redis-resume.keyspace=\${SCHEDULER_REDIS_RESUME_KEYSPACE}
+cdc.redis-resume.fallbackInMin=\${SCHEDULER_REDIS_RESUME_FALLBACK_IN_MIN}
```

Comment:
cdc.redis-resume.fallbackInMin is not needed.

Reply:
done here a963bf41949734565b131774a0da0809fc4bfc61